tests/Backend/MixedSnowflakeBigquery/SharingTest.php

```diff
@@ -198,4 +199,21 @@ public function testAllSharingOperations(): void
         $firstLinked = reset($linkedBuckets);
         $this->assertEquals($bucketId, $firstLinked['sourceBucket']['id']);
     }
+
+    public function testCreateTableInBucketWithDifferentBackendAsProjectDefault(): void
+    {
+        $this->initTestBuckets(self::BACKEND_SNOWFLAKE);
+        $bucketId = $this->getTestBucketId();
+
+        // create table in SHARING bucket in project A
+        $tableId = $this->_client->createTableAsync(
```

Comment:
Bez upravy v connectionto spadne na `No driver client available for backend "snowflake"` ked to volam na BQ mixed suite